services/core-web/src/components/mine/Tailings/MineTailingsInfo.js

```diff
@@ -137,7 +137,6 @@ export class MineTailingsInfo extends Component {
   handleAddTailings = (values) => {
     const payload = {
       ...values,
-      has_itrb: values.has_itrb === "true",
```

Comment:
same goes for payload. You can pass 'values' directly

Reply:
fixed.